ext/sockets/sockets.stub.php

```diff
@@ -2017,13 +2046,13 @@ function socket_read(Socket $socket, int $length, int $mode = PHP_BINARY_READ):
  * @param string $address
  * @param int $port
  */
-function socket_getsockname(Socket $socket, &$address, &$port = null): bool {}
+function socket_getsockname(Socket $socket, &$address, ?int &$objint = null): bool {}
 
 /**
  * @param string $address
  * @param int $port
  */
-function socket_getpeername(Socket $socket, &$address, &$port = null): bool {}
+function socket_getpeername(Socket $socket, &$address, ?int &$objint = null): bool {}
```

Comment:
This is a BC break due to named parameters

Reply:
Do you mean UPGRADING entry ?

Reply:
I just wanted to make sure that you noticed the breaking change: existing code like `socket_getpeername($socket, $address, port: $port)` will stop working.

I'm not sure what our policy is regarding those, but I think that sending an email to internals to check for objections would be worth it.